saq/job.py

```diff
@@ -179,8 +187,10 @@ def _duration(self, a, b):
     def stuck(self):
         """Checks if an active job is passed it's timeout or heartbeat."""
         current = now()
+        # if timeout == None, the if statement will fail for
```

Comment:
Is this a comment for _why_ you added the condition below? Good catch, but I don't think this comment should be in the code itself.

Reply:
Yep, you are correct.  It was a note re: why I made the modification, and it should have been removed.